src/main/java/com/mojang/brigadier/tree/LiteralCommandNode.java

```diff
@@ -27,6 +34,11 @@ public LiteralCommandNode(final String literal, final Command<S> command, final
         this.literal = literal;
     }
 
+    /**
+     * Returns the literal this command is triggered by.
```

Comment:
```suggestion
     * Returns the literal this node represents.
```

Or maybe something else instead of "represents"